SectionedSurface-Aplitop-1/SectionedSurface-Aplitop-1.IFC

```diff
@@ -0,0 +1,87 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition []'),'2;1');
```

Comment:
ViewDefinition can be set to [Ifc4X3NotAssigned] for now.

Reply:
Done